### PR TITLE
Show a simplistic message to help users find apps on the market

### LINKED ISSUE #1497: When user is searching for an app in their personal apps list, if app not found, suggest searching on the app market
Steps to reproduce:

- Sign into a Sandstorm server

- Visit `/apps` by clicking **Apps**

- Type `donuts` into the search box

Expected behavior:

- See some hint that you should do the search on the app market

Actual behavior:

- See a terrifying emptiness

### PR DESCRIPTION
**Pull request by paulproteus**
Close #1497

### CODE REVIEW

**Comment by paulproteus** [comment]
Screenshots.

searching-phase-1

searching-phase-2

Requesting preliminary review of code and strings from @zarvox and @neynah 

Note:

- I wanted to implement this as quickly as possible so I don't fail to. I understand that @neynah had a user experience confusion about what to do next, after typing an app into the apps search within Sandstorm. I have personally witnessed someone else have the same problem, so I am in favor of making any change here, even a small win.

- I think linking people to the app market home page is better than deep-linking them to their same search on our existing app market. Rationale: I think that the app market text search is of low quality, whereas I think visiting the app market and scrolling down a little bit and finding the app you wanted is a pretty high-quality experience.

**Comment by neynah** [comment]
This looks pretty great. Perhaps "app" -> "app installed".

**Comment by paulproteus** [comment]
@kentonv ready for your review/merge now. Thanks to @zarvox & @neynah for the discussion that led to this text; if people have other string fix suggestions, feel free to push to this branch and/or click "edit" on GitHub to make those changes. I don't care specifically what color this bikeshed is painted, but I do care that we have a place to store bikes. Metaphorically speaking.

**Comment by zarvox** [comment]
I'd rephrase (mostly to pluralize):

"You have no apps installed with that name.

{link}Search on the app market &raquo;{/link}."

**Comment by kentonv** [comment]
&lt;bikeshed&gt;I feel like "installed apps" makes more sense than "apps installed"&lt;/bikeshed&gt;

**Comment by paulproteus** [comment]
OK I believe this addresses feedback from @kentonv with hints from @zarvox. Please review/merge!

**Comment by zarvox** [comment]
LGTM, definitely a usability improvement.  Thanks for doing this!

**Comment by kentonv** [comment]
> OK I believe this addresses feedback from @kentonv

It looks like my feedback was not included. Did you mean to? I don't feel strongly about my comment so you're welcome to veto it, but just checking if that's what you intended...

**Comment by paulproteus** [comment]
Hmm! I did intend to integrate your feedback, @kentonv 

The final commit <https://github.com/sandstorm-io/sandstorm/commit/e8b755c4803b1b30a4f874c9c1e4128d9f8a119f> changes the string to:

```
      You have no apps installed with that name. Try
      <a href="{{appMarketUrl}}">searching on the app market &raquo;</a>
```

I took "apps installed" from you and the sentence structure from @zarvox . If I missed a comment of yours, or misunderstood a comment of yours, let me know! I was aiming to be responsive to your (good IMHO) suggestion.

**Comment by zarvox** [comment]
Switch "apps" and "installed" and you'll get what Kenton was after. :)

**Comment by paulproteus** [comment]
Oh! I misread @kentonv's comment. OK! One more commit coming up.

**Comment by paulproteus** [comment]
I believe that takes care of it. @kentonv ready for your review/merge.
